src/Microsoft.Health.Fhir.Core/Features/Resources/Bundle/BundleMetricsNotification.cs

```diff
@@ -6,16 +6,14 @@
 using System.Collections.Generic;
 using Microsoft.Health.Fhir.Core.Features.Metrics;
 using Microsoft.Health.Fhir.Core.Models;
-using Microsoft.Health.Fhir.ValueSets;
-using static Hl7.Fhir.Model.Bundle;
 
-namespace Microsoft.Health.Fhir.Api.Features.Resources.Bundle
+namespace Microsoft.Health.Fhir.Core.Features.Resources.Bundle
 {
     public class BundleMetricsNotification : IMetricsNotification
     {
-        public BundleMetricsNotification(IDictionary<string, int> apiCallResults, BundleType bundleType)
+        public BundleMetricsNotification(IDictionary<string, int> apiCallResults, string bundleType)
```

Comment:
Non-blocking / non-urgent nit: does it make sense to rename "bundleType" --> "fhirOperation"?

Reply:
My thought was to keep it as bundleType to show that this expects a type of bundle as opposed to any type of FHIR operation. The notification still uses the FHIR operation field due to the interface it inherits from, but having a different constructor argument name makes its expectations more clear in my mind.

Reply:
Got it, agree! Thanks for the context.